Vic2ToHoI4/Source/OutHoi4/Leaders/OutCountryLeader.cpp

```diff
@@ -0,0 +1,20 @@
+#include "OutCountryLeader.h"
+
+
+
+void HoI4::outputCountryLeader(std::ostream& output, const HoI4::CountryLeader& leader)
+{
+	output << "create_country_leader = {\n";
+	output << "\tname = \"" << leader.getName() << "\"\n";
+	output << "\tdesc = \"" << leader.getDescription() << "\"\n";
+	output << "\tpicture = \"" << leader.getPicture() << "\"\n";
+	output << "\texpire = \"1965.1.1\"\n";
+	output << "\tideology = " << leader.getIdeology() << "\n";
```

Comment:
Just to confirm, even for misloaded leaders, ideology will always be set?

Reply:
It will now. 😄